elide-async/src/main/java/com/yahoo/elide/async/operation/TableExportOperation.java

```diff
@@ -172,13 +170,11 @@ public AsyncAPIResult call() {
     /**
      * Initialize and get RequestScope.
      * @param exportObj TableExport type object.
-     * @param user User Object.
-     * @param apiVersion API Version.
+     * @param scope RequestScope.
      * @param tx DataStoreTransaction.
      * @return RequestScope Type Object
      */
-    public abstract RequestScope getRequestScope(TableExport exportObj, User user,
-            String apiVersion, DataStoreTransaction tx);
+    public abstract RequestScope getRequestScope(TableExport exportObj, RequestScope scope, DataStoreTransaction tx);
```

Comment:
Did this contract change so that we can extract the requestHeaders and use when creating the new RequestScope